src/main/java/com/graphhopper/converter/api/NominatimEntry.java

```diff
@@ -27,14 +28,39 @@ public NominatimEntry(long osmId, String type, double lat, double lon, String di
         this.displayName = displayName;
 
         this.address = new Address();
-        address.setCountry(country);
-        address.setCity(city);
+        address.country = country;
+        address.city = city;
     }
 
     public NominatimEntry() {
         this.address = new Address();
     }
 
+    public boolean isStreet() {
+        if ("highway".equals(this.classString)) {
+            return true;
+        }
+        return false;
```

Comment:
Not a big deal. You can of course also keep it. For me the shorter is more readable but of course this is discussable ;)